### ISSUE TITLE
poll result bar colour very light

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/adhocracy-plus#2369]
**URL:** https://aplus-dev.liqd.net/sandis/projects/module/umfrage-38/
**user:** all
**expected behaviour:** I can always easily see the result.
**behaviour:** When the poll results are in the wrong angle on my screen, I cannot see the bars and only have the numbers to know which is the most given answer.
**important screensize:**
**device & browser:** 
**Comment/Question:** The bar colour is very light, can we use a darker one @mcastro-lqd ? We would have to check the other places where we use these colours as well, though.

Screenshot?
